### PR TITLE
Nerfs shotguns, buffs lasers

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request
This PR makes the following changes:
Slug damage lowered 60 -> 40
Buckshot damage lowered 75 -> 60
Riot shotgun has a fire_delay of 7
Combat shotgun has a fire_delay of 5
Laser gun damage upped to 30.

## Why It's Good For The Game
Shotguns have slowly power creeped, _hard_. Right now, instead of lugging around a fire axe, you could very easily just carry around a combat shotgun, or even a normal sized sawn off riot gun, for easy 75 melee-range damage, plus the lag from the buckshot hitting you all at once (which i'm aiming to fix in another PR)

This is compounded by the fact that for some reason shotguns didn't have any real fire delay. You could kill people in a literal _second_ with a combat shotgun, whereas with a fire axe you need at least, 6 or 10.

The ammo 'problem' that's supposed to counter that isn't real. Reloading a shotgun is extremely easy due to the four buckshot boxes in the armory, the hacked autolathe, and the massive amounts of shells you can very easily get via the security protolathe. Instead of the ballistics ammo being a downside, to a 60/75 damage weapon, it becomes an upside, allowing you to reload on the go at very minimal cost.

Additionally, combat shotguns are easily orderable en-masse during war-ops, or cult, or other such reasons to hoard every budget card and bloat the cargo budget. This results in the rounds being near-unwinnable by the antagonists as 5 nuke ops or 10 cultists try to beat 70 people that are equally, if not better, equipped than them.

The results of this are that non-shotgun weapons (particularly lasers) are very rarely used in an emergency, at least in my experience. I know I have never even thought of picking up a laser gun to fight a cult, always opting for a shotgun instead. Why would I grab a gun that fires 12 slow shots of 20 damage when I could grab a gun that instantly unloads 7 shells of 75/60 damage on an enemy? At best, I'd settle for twohanding disablers. Lasers guns themselves are not only far weaker than slugs, but weaker than disablers, too. They need 5 shots to drop while disablers need only 4, _and_ they have double the ammo consumption. It's more common to find someone disabling then executing with melee than a standard lasering to death, which I feel is silly. If you're aiming to kill, you should probably use the kill option.

## Changelog
:cl:
balance: Rebalanced shotguns and lasers.
balance: Riot shotguns and combat shotguns now have a fire delay, combat shotties being faster.
balance: Laser guns deal 30 burn damage.
balance: Buckshot now deals 60 damage in total, 10 each pellet, and slugs deal 40.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
lasers should be 25, 30 seems like too much

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 1]
```diff
@@ -56,13 +57,15 @@
...
 	item_state = "shotgun_combat"
+	fire_delay = 5
 	mag_type = /obj/item/ammo_box/magazine/internal/shot/com
 	w_class = WEIGHT_CLASS_HUGE
 
 /obj/item/gun/ballistic/shotgun/automatic/combat/compact
 	name = "compact combat shotgun"
 	desc = "A compact version of the semi automatic combat shotgun. For close encounters."
 	icon_state = "cshotgunc"
+	fire_delay = 5
```
Comment (Fikou) [inline code/modules/projectiles/guns/ballistic/shotgun.dm]: it inherits the fire delay from parent this is uselss

**Comment by optimumtact** [comment]
just make the shotgun a two handed weapon

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 2]
```diff
@@ -1,11 +1,11 @@
 /obj/projectile/bullet/shotgun_slug
 	name = "12g shotgun slug"
-	damage = 60
+	damage = 45
```
Comment (Fikou) [inline code/modules/projectiles/projectile/bullets/shotgun.dm]: the pr says that you cahnged it to 40 but its 45

Comment (carlarctg) [reply code/modules/projectiles/projectile/bullets/shotgun.dm]: whoops

File: code/modules/projectiles/guns/ballistic/shotgun.dm (hunk 1 above)
Comment (carlarctg) [reply code/modules/projectiles/guns/ballistic/shotgun.dm]: heck

**Comment by Dawson1917** [comment]
Like you mentioned, infinitely available ammo from autolathes and sec protolathes and cargoguns entirely invalidates the purpose of lasers and ballistics in general and shotguns in particular. They're meant to be heavy duty weapons that far excel in damage but have a very limited number of uses before you have to throw them away. That's the source of the problem, lowering shotgun damage is only bandaiding the problem. Guncargo and lathe ammo are what need to be stomped so ammunition can mean something. 
Fire delay is a good change. So is the ""soon"" to come two-hand requirement for shotguns. 40dmg slugs is good.

Lasers don't need additional damage. Energy weapons are designed to be, for guns, low-damage with high re-usability thanks to rechargeable batteries. Disablers need their capacity lowered so they aren't infinitely better than lethals at killing people.

**Comment by AdamElTablawy** [comment]
the fire delay is a good change, but compounding fire delay with damage nerfs makes the shotgun kind of mediocre since now not only are you shooting slower but you're not doing as much damage, either

also, lasers doing 30 means they'll rip through strong blobs pretty quickly, and slugs doing just 10 more damage than lasers makes me question why anyone would use slugs at that point since laser guns do similar amounts of damage and would hold more ammo.

all of those changes independently are okay but all together it's a lot

**Comment by zxaber** [comment]
A better fix for ammo being mass printable is requiring more than just metal to make it. Something to account for the gunpowder would be good.

Maybe you could make "unfinished shells" in the autolathe and then get something from chemistry to act as the powder and craft it together?

**Comment by carlarctg** [comment]
> A better fix for ammo being mass printable is requiring more than just metal to make it. Something to account for the gunpowder would be good.
> 
> Maybe you could make "unfinished shells" in the autolathe and then get something from chemistry to act as the powder and craft it together?

calm down there satan

**Comment by Iatots** [comment]
Bullet pipelines when

**Comment by Krysonism** [comment]
Huge overnerf, also 30 damage lasers is too much

**Comment by 81Denton** [comment]
I'm not sold on the damage changes and personally prefer #47039 to this, where shotguns are still powerful but unwieldy. The riot shotgun fire_delay changes are decent though, you can fire riot shotguns almost as fast as combat shotguns if you're quick enough right now.

**Comment by necromanceranne** [comment]
I feel like you weren't considering nonlethal ammunition when balancing these, so as a heads up, beanbags can pop someone into stamina crit in two shots, whereas slugs can't put you into crit with two, even without factoring any armor. I'd recommend that if you're going to reduce damage, give them armor penetration to compensate, so they actually stand apart from their nonlethal counterpart.

Rubbershot compared to buckshot is...roughly the same so there isn't much to be worried about there.

30 damage lasers makes them very, very good when dual-wielded and fired at close range. You go from one problem of someone meatshotting you to death with a shotgun and introduce lasers as an alternative, which you are likely to hit crit very quickly from someone pointblanking you with two lasers. This has been shown already with dual-wielded disablers being the favoured method of some sec players for taking people down, because it's pretty much unanswerable and somewhat invalidates even needing a stun baton. There is a real problem with dual-wielding right now but it isn't really being brought up a whole lot, so here is as good a place to highlight that 'hey man, getting point blanked by some dude going guns akimbo is kind of silly'

Let's not even get into the one-handed burst weapons. That shit is...really fucked up.

**Comment by carlarctg** [comment]
> I feel like you weren't considering nonlethal ammunition when balancing these, so as a heads up, beanbags can pop someone into stamina crit in two shots, whereas slugs can't put you into crit with two, even without factoring any armor. I'd recommend that if you're going to reduce damage, give them armor penetration to compensate, so they actually stand apart from their nonlethal counterpart.
> 
> Rubbershot compared to buckshot is...roughly the same so there isn't much to be worried about there.
> 
> 30 damage lasers makes them very, very good when dual-wielded and fired at close range. You go from one problem of someone meatshotting you to death with a shotgun and introduce lasers as an alternative, which you are likely to hit crit very quickly from someone pointblanking you with two lasers. This has been shown already with dual-wielded disablers being the favoured method of some sec players for taking people down, because it's pretty much unanswerable and somewhat invalidates even needing a stun baton. There is a real problem with dual-wielding right now but it isn't really being brought up a whole lot, so here is as good a place to highlight that 'hey man, getting point blanked by some dude going guns akimbo is kind of silly'
> 
> Let's not even get into the one-handed burst weapons. That shit is...really fucked up.

twohanding laser/disabler guns is another issue thats pretty much unrelated to the point of this pr

if/when it becomes unbearable someone or me will PR a fix for it

**Comment by carlarctg** [comment]
> I'm not sold on the damage changes and personally prefer #47039 to this, where shotguns are still powerful but unwieldy. The riot shotgun fire_delay changes are decent though, you can fire riot shotguns almost as fast as combat shotguns if you're quick enough right now.

yeah, if that PR gets merged i'm removing most nerfs here

**Comment by TheRespectedMan** [comment]
the fire delay hurts allot, even despite the damage nerfs i still view the shotgun as viable but the fire delay persuades me to not use it anymore.

**Comment by CRITAWAKETS** [comment]
Do not forget to buff the autorifle while you are at it. It requires more effort than laser rifles to get and is supposed to be a ballistic version.

**Comment by necromanceranne** [comment]
God forbid the WT550, which is 1200 credits in a single pack, be equal to a stechkin...

**Comment by Naloacisamazing** [comment]
Leave lasers as is and nerf shotties. I see no need to buff lasers

**Comment by PKPenguin321** [comment]
laser buff's real dumb, yeah. lasers are relatively common (especially compared to shotguns) and already drop you in 5 shots, the buff is just nonsense

**Comment by 81Denton** [comment]
Closing this in favor of #47039.

**Pull request closed without merging**